### ISSUE TITLE
_controller.clear() bug.

### ISSUE THREAD

**Issue by KaeganClark** [4Q-s-r-o/signature#19]
_controller.clear() doesn't work when called. The user must call the function and then start drawing again before it clears.

**Comment by dcordi** [comment]
a temporary solution : 

**modify signature.dart.**
add this line after import declaration : 
`final signatureKey = GlobalKey();`

add this line after 'final Color backgroundColor;'
`final Key key = signatureKey ;`

and replace clear() method on SignatureController by : 
```
  clear() {
    value = List<Point>();
    this.notifyListeners();
    signatureKey.currentState.setState(() {
    }) ;
  }
```

Good luck
Daniel

**Comment by SebastienBtr** [comment]
You just need to refresh the view with a "set state":

`setState(() => _controller.clear())`

**Comment by MartinHlavna** [comment]
@SebastienBtr's way is correct way to use this.

Hovewer this is also fixed in 3.1.0
